evalml/tests/automl_tests/test_automl.py

```diff
@@ -1633,15 +1633,15 @@ def test_targets_pandas_data_types_classification(
         )
 
     if automl_type == ProblemTypes.BINARY:
-        X, y = load_breast_cancer()
+        X, y = load_breast_cancer(use_local=True)
         if "bool" in target_type:
             y = y.map({"malignant": False, "benign": True})
     elif automl_type == ProblemTypes.MULTICLASS:
         if "bool" in target_type:
             pytest.skip(
                 "Skipping test where problem type is multiclass but target type is boolean"
             )
-        X, y = load_wine()
+        X, y = load_wine(use_local=True)
```

Comment:
I think it may be best to just create `wine_data`, `breast_cancer_data` fixtures that set `use_local=True`. What do you think?

Reply:
Yeah that makes sense 👍